src/main/java/org/openRealmOfStars/starMap/StarMapGenerator.java

```diff
@@ -0,0 +1,1744 @@
+          starMap.setTile(px + 1, py,
+              Tiles.getTileByName(TileNames.SATURN_NE));
+          starMap.setTile(px, py + 1,
+              Tiles.getTileByName(TileNames.SATURN_SW));
+          starMap.setTile(px + 1, py + 1,
+              Tiles.getTileByName(TileNames.SATURN_SE));
+          break;
+        }
+        case 31: {
+          starMap.setTile(px, py,
+              Tiles.getTileByName(TileNames.ICEGIANT1_NW));
+          starMap.setTile(px + 1, py,
+              Tiles.getTileByName(TileNames.ICEGIANT1_NE));
+          starMap.setTile(px, py + 1,
+              Tiles.getTileByName(TileNames.ICEGIANT1_SW));
+          starMap.setTile(px + 1, py + 1,
+              Tiles.getTileByName(TileNames.ICEGIANT1_SE));
+          break;
+        }
+        case 32: {
+          starMap.setTile(px, py,
+              Tiles.getTileByName(TileNames.ICEGIANT2_NW));
+          starMap.setTile(px + 1, py,
+              Tiles.getTileByName(TileNames.ICEGIANT2_NE));
+          starMap.setTile(px, py + 1,
+              Tiles.getTileByName(TileNames.ICEGIANT2_SW));
+          starMap.setTile(px + 1, py + 1,
+              Tiles.getTileByName(TileNames.ICEGIANT2_SE));
+          break;
+        }
+        default:
+          throw new IllegalArgumentException("Unexpected gas giant type:"
+             + planet.getPlanetTypeIndex());
+        }
+      }
+    }
+  }
+
+  /**
+   * Build Black hole. This will build black hole in the galaxy center.
+   */
+  private void buildBlackHole() {
+    int cx = starMap.getMaxX() / 2;
+    int cy = starMap.getMaxY() / 2;
+    starMap.setTile(cx, cy, Tiles.getTileByName(TileNames.BLACKHOLE_C));
+    starMap.setSquareInfo(cx, cy, SquareInfo.TYPE_BLACKHOLE_CENTER, 0);
+    starMap.setTile(cx - 1, cy - 1,
+        Tiles.getTileByName(TileNames.BLACKHOLE_NW));
+    starMap.setSquareInfo(cx - 1, cy - 1, SquareInfo.TYPE_BLACKHOLE, 0);
+    starMap.setTile(cx, cy - 1, Tiles.getTileByName(TileNames.BLACKHOLE_N));
+    starMap.setSquareInfo(cx, cy - 1, SquareInfo.TYPE_BLACKHOLE, 0);
+    starMap.setTile(cx + 1, cy - 1,
+        Tiles.getTileByName(TileNames.BLACKHOLE_NE));
+    starMap.setSquareInfo(cx + 1, cy - 1, SquareInfo.TYPE_BLACKHOLE, 0);
+
+    starMap.setTile(cx - 1, cy, Tiles.getTileByName(TileNames.BLACKHOLE_W));
+    starMap.setSquareInfo(cx - 1, cy, SquareInfo.TYPE_BLACKHOLE, 0);
+    starMap.setTile(cx + 1, cy, Tiles.getTileByName(TileNames.BLACKHOLE_E));
+    starMap.setSquareInfo(cx + 1, cy, SquareInfo.TYPE_BLACKHOLE, 0);
+
+    starMap.setTile(cx - 1, cy + 1,
+        Tiles.getTileByName(TileNames.BLACKHOLE_SW));
+    starMap.setSquareInfo(cx - 1, cy + 1, SquareInfo.TYPE_BLACKHOLE, 0);
+    starMap.setTile(cx, cy + 1, Tiles.getTileByName(TileNames.BLACKHOLE_S));
+    starMap.setSquareInfo(cx, cy + 1, SquareInfo.TYPE_BLACKHOLE, 0);
+    starMap.setTile(cx + 1, cy + 1,
+        Tiles.getTileByName(TileNames.BLACKHOLE_SE));
+    starMap.setSquareInfo(cx + 1, cy + 1, SquareInfo.TYPE_BLACKHOLE, 0);
```

Comment:
Fixed actually twice. First I removed setTiles which caused no black hole on map. I kept black hole drawing there since no easy way to make method for it.